handbook/engineering/distribution/k8s_admin_custom_policy.md

```diff
@@ -0,0 +1,92 @@
+# How to use a k8s pod security policy to simulate admin restrictions
+
+This describes how to simulate being a Sourcegraph admin with restrictions. ie an admin that can only schedule pods with
+security restrictions applied to the pods.
+
+It is assumed you have a test cluster running in GCP or a similar k8s runtime and you are a cluster admin, ie
+you have super powers in the cluster:
+
+```shell script
+kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user $(gcloud config get-value account)
+```
+
+First we need a pod security policy that matches as close as possible the restrictions at the customer site. For example
+(this is just an example, tweak as necessary):
+
+```yaml
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: foopolicy
```

Comment:
Can we avoid using `foo` here an elsewhere? It makes it seem less realistic. If you were actually going to configure a policy for our cluster, what would you call it?

Reply:
fixed